src/java.desktop/share/classes/javax/swing/plaf/basic/BasicRadioButtonUI.java

```diff
@@ -152,8 +145,6 @@ public synchronized void paint(Graphics g, JComponent c) {
         textRect.x = textRect.y = textRect.width = textRect.height = 0;
 
         Icon altIcon = b.getIcon();
-        Icon selectedIcon = null;
-        Icon disabledIcon = null;
 
```

Comment:
So what's this about ?

Reply:
I reverted this changes

Reply:
why ? .. although it seemed they were unused too, it is just that you didn't mention them anywhere and the title doesn't relate to it.

Reply:
I will try to remove such unused local variables in separate PR